src/type.zig

```diff
@@ -3596,12 +3596,12 @@ pub const Type = extern union {
 
     fn intAbiSize(bits: u16, target: Target) u64 {
         const alignment = intAbiAlignment(bits, target);
-        return std.mem.alignForwardGeneric(u64, (bits + 7) / 8, alignment);
+        return std.mem.alignForwardGeneric(u64, @intCast(u16, (@intCast(u17, bits) + 7) / 8), alignment);
```

Comment:
Inner cast should be `@as`

Reply:
Oh yeah very true because that of course can safely cast.